go.mod

```diff
@@ -74,3 +74,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/pingcap/parser => github.com/bb7133/parser v0.0.0-20200429032743-4494e5f18a8d
```

Comment:
I think you can avoid `replace` by using `go get`?

Reply:
It needs to replace the import packages. Maybe we needn't to it.

Reply:
Sure, thanks.